.github/workflows/ci.yml

```diff
@@ -2,16 +2,15 @@ name: CI
 on:
   push:
     branches:
-      - "actions/**"
-      - "wip/**"
```

Comment:
We don't have any branches that meet these patterns are present. I elected to remove this to avoid spending CI time on anything that isn't either on main or merging to main.

We can reinstate these if and when we have reason to start building long-lived branches in future.